serverless.yml

```diff
@@ -7,13 +7,27 @@ provider:
   runtime: nodejs14.x
 
 functions:
+  addProduct:
```

Comment:
@parabolabam , It should be createProduct as per the task description

Reply:
Well, the task description is rather a good point, but I believe semantically we adding a product, not creating it. Creating product is responsibility of another essence who produces good :)